modules/migration-DNS-considerations.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * migrating_from_ocp_3_to_4/planning-considerations-3-4.adoc
+
+[id="migration-DNS-considerations_{context}"]
+== DNS considerations
+
+The DNS domain of the OpenShift Container Platform 4 target cluster is different from the domain of the OpenShift Container Platform 3 source cluster. By default, applications get new FQDNs after migration.
+
+If you want to preserve the source DNS domain of migrated applications, consider the following two options.
+
+=== Isolating the DNS domain of the target cluster from the clients
+. Place a network element such as an application load balancer or a reverse proxy between the clients and the OpenShift Container Platform 4 cluster.
+
+. Update the original application FQDN (`app1.apps.ocp3.example.com`) in the DNS server to return the IP address of the network device.
+
+. Configure the network device to send requests received for the application in the source domain to the load balancer in the target {product-title} 4 cluster domain.
+
+. Create a wildcard DNS record for the `*.apps.ocp3.example.com` domain that points to the IP address of the load balancer of the source cluster.
+
+. Create a specific DNS record for each application that points to the IP address of the network device in front of the target cluster. A specific DNS record has higher priority than a wildcard record, so no conflict arises when the application FQDN is resolved.
+
+Keep in mind the following:
+
+* The network device must terminate all secure TLS connections. If the connections pass through to the {product-title} 4 load balancer, the FQDN of the target application is exposed to the client, and certificate errors occur.
+
+* The applications must not return links referencing the target cluster domain to the clients. Otherwise, parts of the application might not load or work properly.
+
+=== Setting up the target cluster to accept the source DNS domain
+You can set up the target cluster to accept requests for migrated applications in the DNS domain of the source cluster.
+
+For both non-secure (HTTP) and secure (HTTPS) access, do the following:
+
+. Create a route in the application’s project for the host name that applications used in the source cluster:
++
+[source,terminal]
+----
+$ oc expose svc app1-svc  --hostname app1.apps.ocp3.example.com \
```

Comment:
I do not think we can use these host name examples. Check the OCP doc guidelines.